appnav/src/main/kotlin/io/element/android/appnav/LoggedInFlowNode.kt

```diff
@@ -227,6 +227,10 @@ class LoggedInFlowNode @AssistedInject constructor(
                     override fun onBackClicked() {
                         backstack.pop()
                     }
+
+                    override fun onInviteAccepted(roomId: RoomId) {
+                        backstack.push(NavTarget.Room(roomId))
```

Comment:
Maybe also pop the backstack if the invitation list is empty?

Reply:
This is currently how it's done on iOS. I guess it might be a bit weird if an invite is removed for some remote reason (e.g. you're uninvited) and that causes a navigation while you're looking at the page?

I'll leave it for now but bear it in mind when the design is reviewed to see if we need to change it on both platforms